source/Applications/PointCloudTutorial.md

```diff
@@ -0,0 +1,131 @@
+
+## Introduction
+
+This tutorial describes how to use Zivid SDK to work with [Point Cloud][kb-point_cloud-url] data.
+
+1. [Frame](#frame)
+   1. [Capture](#capture)
+   2. [Load](#load)
+   3. [Visualize](#visualize)
+3.  [Point Cloud](#point-cloud)
+    1. [Get](#get)
+    2. [Copy](#copy)
+    3.  [Transform](#transform)
+
+### Prerequisites
+
+You should have installed Zivid SDK and cloned C++ samples. For more details see [Instructions][installation-instructions-url].
+
+## Frame
+
+### Capture
+
+The ```Zivid::Frame``` can be captured with a camera ([go to source][frame-capture]).
+```cpp
+const auto frame = camera.capture(settings);
+```
+Check [Capture Tutorial][capture-tutorial] for more details.
+
+### Load
+It can also be loaded from a ZDF file ([go to source][frame-from-file]).
+```cpp
+const Zivid::Frame frame = Zivid::Frame(Zivid3D.zdf);
```

Comment:
Should be a string, like "Zivid3D.zdf".
Also consider making it clear that it's a path, like "/path/to/Zivid3D.zdf".

Reply:
done